archiver/ena.py

```diff
@@ -150,10 +150,10 @@ def convert_sequencing_run(hca_data: dict):
     mapper = JsonMapper(hca_data)
     converted_data = mapper.map({
         '$on': 'process',
-        'alias': ['uuid.uuid', prefix_with, _sq_run_alias_prefix],
+        #being overwritten 'alias': ['uuid.uuid', prefix_with, _sq_run_alias_prefix],
         'title': ['content.process_core.process_name', default_to, ''],
         'description': ['content.process_core.process_description', default_to, ''],
-        'assayRefs': ['uuid.uuid', _sq_run_assay_ref]
+        #being overwritten 'assayRefs': ['uuid.uuid', _sq_run_assay_ref]
```

Comment:
comments cleanup

Reply:
I think it's worth keeping these in until we review the how the conversion mapping being uses
The ENA sequencing runs are essentially mapped 2-3 times,
1. firstly [here](https://github.com/ebi-ait/ingest-archiver/blob/2e3e0e23312323752d159a8e3cbd2a2a0452e1d6/archiver/archiver.py#L957) when the archiver assembles the entity.
1. secondly in the commented section using the conversion map
1. Then [here](https://github.com/ebi-ait/ingest-archiver/blob/2e3e0e23312323752d159a8e3cbd2a2a0452e1d6/archiver/archiver.py#L733), the id and links elements are discarded in favour of the results from the first conversion

Thats quite a subtle series of events, perhaps this warrants a technical debt ticket?

Reply:
Assay refs aren't part of the conversion atm, this should actually be removed.